extensions/avro/runtime/src/main/java/org/apache/camel/quarkus/component/avro/AvroSchemaRegistrySubstitution.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.avro;
+
+import io.quarkus.runtime.ObjectSubstitution;
+import org.apache.commons.lang3.SerializationUtils;
+
+/**
+ * {@code AvroSchemaRegistrySubstitution} offers a way to bypass Quarkus
+ * default serialization. This workaround has been introduced as Quarkus default
+ * serialization used to failed with {@code org.apache.avro.Schema}.
+ */
+public class AvroSchemaRegistrySubstitution implements ObjectSubstitution<AvroSchemaRegistry, byte[]> {
+
+    @Override
+    public byte[] serialize(AvroSchemaRegistry registry) {
+        return SerializationUtils.serialize(registry);
```

Comment:
SerializationUtils.(de)serialize() seem to be the only methods used from commons-lang3. Inlining these methods (which should go quite easily) would allow us to eliminate the commons-lang3 dependency and thus reduce number of classes GraalVM has to analyze and eo ipso speed up the native image builds.

Reply:
Should be ok now.